spec/decorators/course_decorator_spec.rb

```diff
@@ -243,6 +243,15 @@
 
         it { is_expected.to eq("Student finance if you’re eligible") }
       end
+
+      context "Courses excluded from bursaries" do
+        let(:mathematics) { build(:subject, :mathematics, bursary_amount: "3000") }
+        let(:pe) { build(:subject) }
+
+        let(:course) { build :course, name: "PE with mathematics", subjects: [mathematics, pe] }
+
+        it { is_expected.to eq("Bursary or student finance if you’re eligible") }
```

Comment:
something just dont read right

"Courses excluded from bursaries" it  is expected to eq Bursary

Reply:
Yep my bad.

See https://github.com/DFE-Digital/manage-courses-frontend/pull/777/commits/f8c32de565f2449198e29cc1082eafe5edf39b3d